src/network/network.cpp

```diff
@@ -11,16 +11,23 @@ namespace Network {
 
 static std::shared_ptr<RoomMember> g_room_member; ///< RoomMember (Client) for network games
 static std::shared_ptr<Room> g_room;              ///< Room (Server) for network games
+static bool initialized = false;                  ///< Network initialization state
+static bool in_room = false;                      ///< Track if we're currently in a room
 // TODO(B3N30): Put these globals into a networking class
 
 bool Init() {
+    if (initialized) {
+        return true;
+    }
+
     if (enet_initialize() != 0) {
-        LOG_ERROR(Network, "Error initalizing ENet");
```

Comment:
Why removing that log?

Reply:
Accidentally

Mind adding it back?

Reply:
I can add back myself np, ill start improving the code